andela/travel_tool_front
# #161583616 Fix redirect url on modal close

#### What does this PR do?
This PR fixes the error while redirecting to different pages on the app from the request modal

#### Description of Task to be completed?
- fix redirect url on modal close

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Enter into the project directory - cd travel_tool_front && git checkout bg-fix-modal-redirect-161583616
- Install dependencies and start the app - make start
- Start the backend. git clone https://github.com/andela/travel_tool_front.git && make start
- Navigate to travela in your browser and log onto the app.
- Click the created request either on `Approvals` or `Requests` page
- Close the modal you should still remain on the page in which you click the request 

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#161583616](https://www.pivotaltracker.com/story/show/161583616)

#### Screenshots (if appropriate)


#### Questions:
